brand_external_report_layout/models/res_brand.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+import os
+from odoo import api, fields, models, tools, _
+
+
+class ResBrand(models.Model):
+
+    _inherit = 'res.brand'
+
+    def _get_logo(self):
+        return base64.b64encode(
+            open(
+                os.path.join(
+                    tools.config['root_path'],
+                    'addons',
+                    'base',
+                    'static',
+                    'img',
+                    'res_company_logo.png',
+                ),
+                'rb',
+            ).read()
+        )
+
+    logo = fields.Binary(
+        related='partner_id.image',
+        default=_get_logo,
```

Comment:
Do we need a default? As it is a related, does it not inherit the default from `partner_id.image`?

Reply:
the default partner image is a person avatar, I changed by the default company logo.

person avatar: ![avatar](https://user-images.githubusercontent.com/12665409/68990608-68b29d80-0855-11ea-995e-301c6760085c.png)

company logo:  ![res_company_logo](https://user-images.githubusercontent.com/12665409/68990617-86800280-0855-11ea-9f56-1eb8cc5fd49f.png)